index.html

```diff
@@ -359,6 +359,56 @@ <h4 class="color-black text-20">Skillsets</h4>
           </div>
         </div>
       </section>
+
+      <!-- Contact form -->
+      <section class="contact-form-section flex-column">
+        <div class="contact-form-bg"></div>
+        <article class="flex-column">
+          <h3 class="text-40 color-white text-align-center">Contact me</h3>
+          <p class="text-20-LH-28 color-B50 text-align-center">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it!
+          </p>
+        </article>
+        <article>
+          <form
+            class="form-div flex-column"
+            action="https://formspree.io/f/xoqzkzzr"
+            method="post"
+          >
+            <input
+              class="text-input color-N800 text-17-LH-20"
+              name="username"
+              id="username"
+              type="text"
+              maxlength="30"
+              placeholder="Lorem Ipsum"
+            />
+            <input
+              class="email-input color-N800 text-17-LH-20"
+              name="Email"
+              id="email"
+              type="email"
+              maxlength="30"
+              placeholder="loremipsum@gmail.com"
+              required
+            />
+            <textarea
+              class="textarea-input color-N60 text-15-bold"
+              name="message"
+              id="message"
+              type="textarea"
+              placeholder="write a new message here..."
+              maxlength="500"
+              rows="5"
+            ></textarea>
```

Comment:
- [ ] Although you have done a commendable job :+1:, I kindly request that you make a small addition to meet the project requirements. Specifically, it would be beneficial if you added the attribute 'required' to both the name input and the text area. This will ensure that users cannot submit the form without filling out these fields, and improve the overall usability and reliability of the form.

Reply:
oh yes, thank you